doc/source/autoscaling.rst

```diff
@@ -40,6 +40,57 @@ Test that it works by running the following commands from your local machine:
 
 .. note:: You may see a message like: ``bash: cannot set terminal process group (-1): Inappropriate ioctl for device bash: no job control in this shell`` This is a harmless error. If the cluster launcher fails, it is most likely due to some other factor.
 
+Azure
+~~~~~
+
+First, install the Azure CLI (``pip install azure-cli azure-core``) then login using (``az login``). 
+
+Set the subscription to use from the command line (``az account set -s <subscription_id>``) or by modifying the provider section of the config provided e.g: `ray/python/ray/autoscaler/azure/example-full.yaml`
+
+Once the Azure CLI is configured to manage resources on your Azure account, you should be ready to run the autoscaler. The provided `ray/python/ray/autoscaler/azure/example-full.yaml <https://github.com/ray-project/ray/tree/master/python/ray/autoscaler/azure/example-full.yaml>`__ cluster config file will create a small cluster with a Standard DS2v3 head node (on-demand) configured to autoscale up to two Standard DS2v3 `spot workers <https://docs.microsoft.com/en-us/azure/virtual-machines/windows/spot-vms>`__. Note that you'll need to fill in your resource group and location in those templates.
+
+Test that it works by running the following commands from your local machine:
+
+.. code-block:: bash
+
+    # Create or update the cluster. When the command finishes, it will print
+    # out the command that can be used to SSH into the cluster head node.
+    $ ray up ray/python/ray/autoscaler/azure/example-full.yaml
+
+    # Get a remote screen on the head node.
+    $ ray attach ray/python/ray/autoscaler/azure/example-full.yaml
+    $ source activate tensorflow_p36
+    $ # Try running a Ray program with 'ray.init(address="auto")'.
+
+    # Tear down the cluster.
+    $ ray down ray/python/ray/autoscaler/azure/example-full.yaml
+
+Azure Portal
+~~~~~~~~~~~~
+
+Alternatively, you can deploy a cluster using Azure portal directly. Please note that auto scaling is done using Azure VM Scale Sets and not through
+Rays auto-scaler. This will deploy `Azure Data Science VMs (DSVM) <https://azure.microsoft.com/en-us/services/virtual-machines/data-science-virtual-machines/>`_
```

Comment:
```suggestion
Ray's autoscaler. This will deploy `Azure Data Science VMs (DSVM) <https://azure.microsoft.com/en-us/services/virtual-machines/data-science-virtual-machines/>`_
```